CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+v2.14.1
+=======
+* fixed incorrect decoding `content-type` from header
```

Comment:
Consider adding more information to changelog. (Now, it's not obvious for the end users should they migrate SAR version in their project or not. In some cases it is not necessary, eg. when all requests to the "SAR endpoints" in some app contains only 'application/json' as `content-type`).

Example:
```md
* Added support for `content-type` header containing additional information like `encoding=UTF-8`. Previously, only `application/json` was accepted as `content-type` for request body filters.
 * From now on, headers values with additional informations like `content-type=application/json;encoding=UTF-8` are successfully accepted too. 
```